docs/weave/basics/01-blockchain.md

```diff
@@ -0,0 +1,25 @@
+
+## General Purpose Computer
+
+Ethereum pioneered the second generation of blockchain, where they realized that we didn't have to limit ourselves to handling payments, but actually have a general purpose state machine. They wanted to allow experimentation at a rate orders of magnitude faster than forking bitcoin, and produced the EVM (Ethereum Virtual Machine) that can run sandboxed code uploaded by any user. Since then, hundred of projects have experimented with porting other types of logic to the blockchain, and have demonstrated its utility for [decentralized governance](https://aragon.one/), [currency trading](https://0xproject.com/), [prediction markets](https://gnosis.pm/), even [collectible trading games](https://www.cryptokitties.co/) and much more...
+
+While Ethereum demonstrated the potential of blockchain technology in many areas, we it also provided some [high profile examples](https://www.cryptocompare.com/coins/guides/the-dao-the-hack-the-soft-fork-and-the-hard-fork/) of how [hard it is to write secure contracts](https://medium.com/chain-cloud-company-blog/parity-multisig-hack-again-b46771eaa838) . As it became more popular, it also showed a popular application can [overload the capacity of the network](https://dealbreaker.com/2017/12/ethereum-the-crypto-network-that-will-transform-everything-struggles-to-handle-digital-beanie-babies/) .
+
+## Next Generation
+
+Since that time, many groups are working on "next generation" solutions that take the learnings of Ethereum and attempt to build a highly scalable and secure blockchain that can run general purpose programs. One pioneering project is [Tendermint](https://tendermint.com/), which provides a highly efficient, Byzantine Fault Tolerant blockchain engine offering guaranteed finality in the order of 1-5 seconds. It was designed from the ground up to allow many projects to easily [plug their application logic](https://tendermint.readthedocs.io/en/master/app-development.html#abci-design) into the engine. [Weave](https://github.com/iov-one/weave) is a framework that provides many common tools to help you build ABCI apps rapidly. You can just focus on writing the application logic and the interface and rely on high quality and extensible libraries to solve most of the difficult problems with building a blockchain.
```

Comment:
```suggestion
Since that time, many groups are working on "next generation" solutions that take the learnings of Ethereum and attempt to build a highly scalable and secure blockchain that can run general-purpose programs. One pioneering project is [Tendermint](https://tendermint.com/), which provides a highly efficient, Byzantine Fault Tolerant blockchain engine offering guaranteed finality in the order of 1-5 seconds. It was designed from the ground up to allow many projects to easily [plug their application logic](https://tendermint.readthedocs.io/en/master/app-development.html#abci-design) into the engine. [Weave](https://github.com/iov-one/weave) is a framework that provides many common tools to help you build ABCI apps rapidly. You can just focus on writing the application logic and the interface and rely on high-quality and extensible libraries to solve most of the difficult problems with building a blockchain.
```